src/scripts/main.js

```diff
@@ -1 +1,106 @@
 'use strict';
+
+const createElementDiv1 = document.createElement('div');
+const createElementDiv2 = document.createElement('div');
+const createElementDiv3 = document.createElement('div');
+const createElementDiv4 = document.createElement('div');
+
+createElementDiv1.setAttribute('data-qa', 'notification');
+createElementDiv2.setAttribute('data-qa', 'notification');
+createElementDiv3.setAttribute('data-qa', 'notification');
+createElementDiv4.setAttribute('data-qa', 'notification');
+
+// First Promise
+const firstPromise = new Promise((resolve, reject) => {
+  document.addEventListener('click', () => {
+    resolve('First promise was resolved');
+  });
+
+  setTimeout(() => {
+    reject(new Error('First promise was rejected'));
+  }, 3000);
+});
+
+// Second Promise
+const secondPromise = new Promise((resolve) => {
+  // eslint-disable-next-line no-shadow
+  document.addEventListener('click', function (event) {
+    if (event.button === 0 || event.button === 2) {
+      resolve('Second promise resolved after either left or right click');
+    }
+  });
+
+  document.addEventListener('contextmenu', function (e) {
+    e.preventDefault();
+    resolve('Second promise resolved after either left or right click');
```

Comment:
The message 'Second promise resolved after either left or right click' is misleading. Consider updating the message to accurately reflect the behavior, as it resolves on either left or right click.